community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/MultipleIndexPopulator.java

```diff
@@ -311,18 +317,17 @@ public void flipAfterPopulation()
 
     private int[] propertyKeyIds()
     {
-        return populations.stream().flatMapToInt( this::propertyKeyIds ).distinct().toArray();
+        return populations.keySet().stream().flatMapToInt( this::propertyKeyIds ).distinct().toArray();
     }
```

Comment:
That is true, but this is a place where any solution works because it's called once per node scan.